srcpkgs/sview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+archs="x86_64"
```

Comment:
Why?

Reply:
Not sure I understand the question...

Why this app? Because I am using it with 3D Ready DLP projector to convert SBS or OU videos to appropriate format.

Why only x86_64? Because this is what I tested, however potentially this could support i686 and ARM.

Reply:
If the software doesn't have any explicit architecture limitiations, don't limit it in `archs`.

Reply:
For now software successfully compiles for x86_64 and i686, there are few issues which I hope you can help me to address:

1. arm\aarch compiler fails on finding `livavcodec.h` headers file. It seems like headers files are not part of the libavcodec package for arm\aarch, so I'm not sure how to deal with that.

2. musl version is failing on attempt to import `xlocate.h` file which is part of the `glibc`. I'm not sure what exactly I need to install for musl to make it work.

Appreciate any help with resolving those questions.

Reply:
1. https://github.com/gkv311/sview/blob/master/Makefile#L57 `make_build_args="FFMPEG_ROOT=${XBPS_CROSS_BASE}/usr/include` or just `/usr`, not sure.

2. patch https://github.com/gkv311/sview/blob/8be57493b851b8172dcab097fea19e18c46fcc37/include/StStrings/StStringStream.h#L23 to `__linux__`, perhaps? That header is weird.